### PR TITLE
Put keys in /opt/lightup

### PR DESCRIPTION
**Pull request by jeff-lightup**
Make sure that keys aren't deleted if lupmgr is replaced

### CODE REVIEW

**Comment by jeff-lightup** [comment]
Still needs to be tested

**Comment by jeff-lightup** [comment]
Tested with a fresh deployment to make sure all worked as expected

File: connect_dataplane_to_lightup.sh [hunk 1]
```diff
@@ -34,16 +33,16 @@ if [[ ${reverse_port_list-} ]]; then
     echo "#!/usr/bin/env bash" > generated/lightup-dataplane-connect-service.sh
     echo  \
     "AUTOSSH_DEBUG=1 \
-    AUTOSSH_LOGFILE=${this_dir}/lightup-dataplane-connect.service.log \
-    AUTOSSH_PIDFILE=${this_dir}/lightup-dataplane-connect.service.pid \
+    AUTOSSH_LOGFILE=/opt/lightup/lightup-dataplane-connect.service.log \
+    AUTOSSH_PIDFILE=/opt/lightup/lightup-dataplane-connect.service.pid \
```
Comment (rajivramanathan) [inline connect_dataplane_to_lightup.sh:37]: Will we have permissions to write it here?

**Review by rajivramanathan — APPROVED** [review]
I think if move over to ngrok. This may not be needed but for completeness.

File: connect_dataplane_to_lightup.sh (hunk 1 above)
Comment (jeff-lightup) [reply connect_dataplane_to_lightup.sh:37]: yup, this has been tested and works

**Pull request merged**